### ISSUE TITLE
Multiple Jasmine Expectations are swallowed by WebdriverIO

### ISSUE THREAD

**Issue by nicholasbailey** [webdriverio/webdriverio#3416]
**Environment (please complete the following information):**
 - **WebdriverIO version:** [e.g. 5.3.3]
 - **Mode:** WDIO Test Runner
 - **If WDIO Testrunner, running sync/async:** sync
 - **Node.js version:** 8.11.3
 - **NPM version:** 5.6.0
 - **Browser name and version:** N/A
 - **Platform name and version:** MacOS High Sierra
 - **Additional wdio packages used (if applicable):** @wdio/jasmine-framework reporter, @wdio/selenium-standalone service]

**Config of WebdriverIO**
```
{
    framework: 'jasmine',
    reporters: ['spec'],
    jasmineNodeOpts: {
        defaultTimeoutInterval: 60000,
    },
}
```

**Describe the bug**
When running a Jasmine test normally, each expect is a soft-assert, in the sense that test execution continues and all failures for a given test case are aggregated at the end. When running Jasmine through the wdio test runner, test execution continues, but only the initial error is reported. 

**To Reproduce**
Set up a project with this spec-file and the wdio config from above.

spec.js
```
describe('expect', () => {
    it('should run all assertions not just the first', () => {
        expect(2).toBe(3);
        expect("a").toBe("b");
    })
})
```

Run wdio against the config file

**Expected behavior**
You would expect to see two error messages, like this:
![screen shot 2019-01-24 at 2 58 57 pm](https://user-images.githubusercontent.com/24800147/51711524-f8027200-1ff1-11e9-87ed-01e991be2598.png)

Instead, you will only see the first error.

**Additional context**
Not sure if this is really more of a bug or a feature request. Right now when running with Jasmine, WebdriverIO only ever reports the first failure. This makes sense, since the test objects used by the various reporters all work with a single 'error' property, which holds a single error object. It would be nice to support the concept of soft assertions, whether in Jasmine or any other framework, by being able to work with a test returning more than one error.

**Comment by istvandesign** [comment]
With mocha I can use multiple expect statements.

**Comment by christian-bromann** [comment]
I think this is fixed now by #3669
